### PR TITLE
Niamh25/doc 5381/api bullet points

### PR DESCRIPTION
**Pull request by Niamh25**
Changed most of the API sections to bullet points for easy reading.

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM
R U sure relref cannot be used here ?

**Comment by nadinequinr3** [comment]
> LGTM R U sure relref cannot be used here ?

Markdown links won't apply code font or open in a new tab.

File: content/en/platform/corda/5.0/developing-applications/api/application/marshalling.md [hunk 1]
```diff
@@ -9,4 +9,6 @@ menu:
 section_menu: corda5
 ---
 
-The `marshalling` package provides services for working with parameters input over the REST API and generating suitable output to return via the REST API. The main service is <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/marshalling/JsonMarshallingService.html" target="_blank">`JSONMarshallingService`</a>, which allows you to work with JSON input and output data.
+The `marshalling` package provides services for working with parameters input over the REST API and generating suitable output to return via the REST API. The main service is:
+
+* The <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/marshalling/JsonMarshallingService.html" target="_blank">`JSONMarshallingService`</a>, allows you to work with JSON input and output data.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/developing-applications/api/application/marshalling.md:14]: I think the Style Guide says lists must have at least 2 bullets.

File: content/en/platform/corda/5.0/developing-applications/api/application/membership.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ menu:
 section_menu: corda5
 ---
 
-The `membership` package provides services for working with membership groups. The <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/membership/MemberLookup.html" target="_blank">`MemberLookup`</a> service allows a flow to discover what counterparties are available in the membership group or retrieve full details of a counterparty with a given name.
+The `membership` package provides the following services for working with membership groups:
+
+* The <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/membership/MemberLookup.html" target="_blank">`MemberLookup`</a> service allows a flow to discover what counterparties are available in the membership group or retrieve full details of a counterparty with a given name.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/developing-applications/api/application/membership.md:14]: I think the Style Guide says lists must have at least 2 bullets.

File: content/en/platform/corda/5.0/developing-applications/api/application/messaging.md [hunk 3]
```diff
@@ -9,6 +9,8 @@ menu:
 section_menu: corda5
 ---
 
-The `messaging` package provides services and types for creating and working with peer-to-peer sessions. The <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/messaging/FlowMessaging.html" target="_blank">`FlowMessaging`</a> service allows you to create new sessions with counterparties. Once created, a <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/messaging/FlowSession.html" target="_blank">`FlowSession`</a> can be used to send and receive messages from a peer.
+The `messaging` package provides services and types for creating and working with peer-to-peer sessions:
+
+* The <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/messaging/FlowMessaging.html" target="_blank">`FlowMessaging`</a> service allows you to create new sessions with counterparties. Once created, a <a href="../../../../../../api-ref/corda/5.0-beta/java/net/corda/v5/application/messaging/FlowSession.html" target="_blank">`FlowSession`</a> can be used to send and receive messages from a peer.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/developing-applications/api/application/messaging.md:14]: I think the Style Guide says lists must have at least 2 bullets.

**Review by nadinequinr3 — DISMISSED** [review]

**Pull request merged**